File: programme/square-root-of-number/SquareRootOfNumber.java
// Write a programme to find square root of a number

import java.util.Scanner;

public class SquareRootOfNumber {
  
  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
    System.out.print("Input : ");
    double input = sc.nextDouble();
    System.out.println("Output : " + squareRootOfNumber(input));
    sc.close();
  }
  
  private static double squareRootOfNumber(double num) {
    return Math.sqrt(num);
  }
}

File: programme/square-root-of-number/square-root-of-number.cs
// Write a programme to find square root of a number
using System;

namespace Codinasion.Programme
{
    public class SquareRootOfNumber
    {
        public static void Main()
        {
            Console.Write("Input  : ");
            String input = Console.ReadLine();
            int x = Int32.Parse(input);
            Console.WriteLine("Output : " + Math.Sqrt(x));
        }
    }
}
